Misc/NEWS.d/next/Library/2019-08-27-03-57-25.bpo-37958.lRORI3.rst

```diff
@@ -0,0 +1,2 @@
+Added the pstats.Stats.get_profile_dict() method to return the profile
```

Comment:
@gpshead I just realized that I never updated the function name from from `get_profile_dict` to `get_stats_profile` after refactoring things from a dict to a dataclass. 

Should I submit another PR that updates this file?